test/fixtures/websockets.html

```diff
@@ -0,0 +1,14 @@
+<div id='result'></div>
+<div id='errorTime'></div>
+<script>
+  // This should get blocked by adblock if it is on
+  var exampleSocket = new WebSocket("ws://ag.innovid.com/dv/sync?tid=2");
```

Comment:
can we use webpack for the blocked ws to prevent a false positive for the error?